kura/examples/bnd_examples/org.eclipse.kura.demo.bnd.heater/pom.xml

```
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <!--
 
     Copyright (c) 2011, 2024 Eurotech and/or its affiliates and others
```

Hi there!

Thank you for spotting this. Would you mind updating the copyright header with the correct date and adding yourself as contributor? There's also a couple of extra whitespaces that throw the automation off.

```suggestion
    Copyright (c) 2011, 2025 Eurotech and/or its affiliates and others
```